src/app/containers/Timestamp/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { number } from 'prop-types';
+import Timestamp from '../../components/Timestamp';
+
+const formatTimestamp = timestamp => {
+  const dateTime = new Date(timestamp * 1000);
+  const day = dateTime.toLocaleString('en-GB', {
+    day: 'numeric',
+  });
+  const month = dateTime.toLocaleString('en-GB', {
+    month: 'long',
+  });
+  const year = dateTime.toLocaleString('en-GB', {
+    year: 'numeric',
+  });
+
+  return `${day} ${month} ${year}`;
+};
+
+const TimestampContainer = ({ timestamp }) => {
```

Comment:
This container could be removed if we move all the logic to the component. The logic is fairly simple so could just be kept in the timestamp component.

Reply:
happy to remove the container